track.go

```diff
@@ -99,6 +101,12 @@ func (t *Track) Read(b []byte) (n int, err error) {
 
 // ReadRTP is a convenience method that wraps Read and unmarshals for you
 func (t *Track) ReadRTP() (*rtp.Packet, error) {
+	if t.cachePacket != nil {
```

Comment:
Can you move this into `Read`, then should be good!

Reply:
What needs to be cached here is the rtp.Packet structure, so why move to Read?

Reply:
This fix only works if the user calls `ReadRTP` if a user calls `Read` they will not get the first packet.

If we move the caching into `Read` then `Read` and `ReadRTP` will be fixed!

Reply:
Oh, I see. So it seems that this is not a good way.